### PR TITLE
Draft meeting minutes of 2022-08-11 (no. 64)

### PR DESCRIPTION
**Pull request by sthagen**
- [x] Draft before meeting (in line with tentative agenda)
- [x] Draft after the meeting

### CODE REVIEW

**Review by dmk42 — APPROVED** [review]

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Voting Member           |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Gerald     | Sullivan  | Micro Focus       | Member                  |
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:12]: Gerald -> Jerry

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+* ACTION on Thanassis to present runtime-specific data, not August 11 but next one, the 25th  
+  * ONGOING (not yet due)
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Gerald Sullivan will receive voting privileges
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:60]: Gerald -> Jerry

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+# 4. Discussion
+
+## 4.1 Review status on finalizing SARIF 2.1 errata [List of next steps - github issue #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+* Michael: Some changes in support from the side of Microsoft but we will move on (docs support)
+* David: Kindly asks to convey thanks to Courtny for doing such a great job.
+
+## 4.2 Review current state of ecosystem ongoing work
+
+* Mary: Working on better support in Visual Studioo for general SARIF support
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:103]: Studioo -> Studio

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+* Mary: POC requested for result source efforts
+* Mary: Working on fixes
+
+## 4.3 Discussion of updates to SARIF web site [https://www.sarif.info](https://www.sarif.info)
+
+* Mary: We did update teh website
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:109]: teh -> the

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+* All: Walk through the website
+* Mary: Includes renaming of collaborators that changed name (Whitesource is now Mend), added Java links, etc
+* David: Wikipedia reviewers have been rejecting the current draft because many links point to source code sites and it seems that would require to much effort for reviewers to vet the relation
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:112]: to much -> too much

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+* Michael: Assumes the references mostly point to github source code sites instead of OASIS links.
+* Pavel: Proposes in terms of process to follow the reviewer suggestsion to have an undebated version publicly available and add the removed content in different form later
+* Michael: Is convinced that we may have misinterpreted the reviewer feedback
+* David: Kindly asks for additioanl references
+* Michael: requests an action on him vor that
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:117]: vor -> for

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+* Mend members met with Chris Meyer to support completing the auto-fix scenario
+
+## 4.5 Discuss end-to-end results management (including code insights protocol)
+
+None
+
+## 4.6 Roadmap review
+
+* Michael: Suggests we review the existing roadmap and move on to further specific topics
+* Michael: Internal discussions le participated in focus mostly on security related analysis
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:130]: le?

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+* Michael: Likes to ask the group if there is interest and support for exploring security related topics
+* Stefan: Suggests to explore our possible contributions to an actionable security ecosystem as the security tech offering TCs to him look more like "lmany fences, only few farmers"
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:132]: lmany -> many

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+* Stefan: An example https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=csaf where a lot of integration and extension has been achieved including CVE, CPE, TLP, VEX, SBOM (spdx, cyclonedx, swid)
+* Michael: Some links associated to the suggested partners, existing work, etc.:
+  * https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=cti-cybox
+  * https://oasis-open.github.io/cti-documentation/stix/intro.html
+  * https://github.com/CybOXProject/schemas/blob/master/objects/Win_Executable_File_Object.xsd#L612
+* Michael: Three main dimensions he sees as important in the security arena and aligned with our existing and possible future contributions:
+  * Raw telemetry
+  * Standards for analysing the data and reaching conclusions
+  * How to deidentify the data for sharing or long term use
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:141]: or -> for

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+  * https://github.com/CybOXProject/schemas/blob/master/objects/Win_Executable_File_Object.xsd#L612
+* Michael: Three main dimensions he sees as important in the security arena and aligned with our existing and possible future contributions:
+  * Raw telemetry
+  * Standards for analysing the data and reaching conclusions
+  * How to deidentify the data for sharing or long term use  
+* Jeff: 
+  * The data has to come together - for now SARIF cannot provide it yet, but it seems to be possible to align a slightly extended SARIF with surrounding devices to achieve some of it
+  * To summarize: SBOM looks good, SARIF looks good, but these are more ornaments and not yet the whole tree the security industry needs
+  * Integrations points?
+* Stefan: Sees only possible way to go per extension / integration points but we should pla twice the effort: One part to come up with suggestions and the other to defend the simplicity of such "intrusion" points
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:146]: pla?

File: meeting_minutes/220811_SARIF_TC_64.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+* Jeff: Thinks SBOM was already touched at the end of the previous section
+* Michael: Kindly asks if Stefan could transform his decade long experience at OASIS into a short writeup who would be probably relevant as contact partners
+* Jeff: Might already have such similar lists / overviews but thinks these lists might be short
+* Nathan: 
+  * Open Telemetry already defines the relationships already that could also be a contact point for that list (cf. eg. https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#source-code-attributes)
+  * Mostly any offering that already offers graphs to navigate the information and relations (cf. eg. https://docs.microsoft.com/en-us/graph/overview)
+* Michael: The tool owners in this TC should all be interested in having devices to provide vulnerabilities in a connected world (example credential exposure, how to actionably support such incidents / findings)
+* Jeff: Interpretability for consumers of the security information
+* Stefan:
+* David: Kindly asks Stefan to put the list ito a github page to also foster future discussion
```
Comment (katrinaoneil) [inline meeting_minutes/220811_SARIF_TC_64.md:162]: ito -> into

**Review by katrinaoneil — APPROVED** [review]
Approved with minor suggestions

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 1 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:12]: The registered name is Gerald and this is what we note here.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 2 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:60]: Registered names.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 4 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:109]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 5 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:112]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 6 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:117]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 7 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:130]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 9 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:141]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 10 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:146]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 11 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:162]: Fixed, thanks.

File: meeting_minutes/220811_SARIF_TC_64.md (hunk 8 above)
Comment (sthagen) [reply meeting_minutes/220811_SARIF_TC_64.md:132]: Fixed, thanks.

**Pull request merged**